dotnet/WPF/VisualLayerIntegration/VisualLayerIntegration/BarGraphUtility/BarGraph.cs

```diff
@@ -244,108 +266,108 @@ public void DrawText(WindowRenderTarget renderTarget, string titleText, string x
 
             var factoryDWrite = new SharpDX.DirectWrite.Factory();
 
-            textFormatTitle = new TextFormat(factoryDWrite, "Segoe", baseTextSize * 5 / 4)
+            _textFormatTitle = new TextFormat(factoryDWrite, "Segoe", baseTextSize * 5 / 4)
             {
                 TextAlignment = TextAlignment.Center,
                 ParagraphAlignment = ParagraphAlignment.Center
             };
-            textFormatHorizontal = new TextFormat(factoryDWrite, "Segoe", baseTextSize)
+            _textFormatHorizontal = new TextFormat(factoryDWrite, "Segoe", baseTextSize)
             {
                 TextAlignment = TextAlignment.Center,
                 ParagraphAlignment = ParagraphAlignment.Far
             };
-            textFormatVertical = new TextFormat(factoryDWrite, "Segoe", baseTextSize)
+            _textFormatVertical = new TextFormat(factoryDWrite, "Segoe", baseTextSize)
             {
                 TextAlignment = TextAlignment.Center,
                 ParagraphAlignment = ParagraphAlignment.Far
             };
 
             renderTarget.AntialiasMode = AntialiasMode.PerPrimitive;
-            renderTarget.TextAntialiasMode = TextAntialiasMode.Cleartype;
+            renderTarget.TextAntialiasMode = SharpDX.Direct2D1.TextAntialiasMode.Cleartype;
 
-            textSceneColorBrush = new SolidColorBrush(renderTarget, black);
+            _textSceneColorBrush = new SolidColorBrush(renderTarget, _black);
 
             var ClientRectangleTitle = new RectangleF(0, 0, textWidth, textHeight);
             var ClientRectangleXAxis = new RectangleF(0,
                 containerHeight - textHeight + sgOffsetY * 2, textWidth, textHeight);
             var ClientRectangleYAxis = new RectangleF(-sgOffsetX,
                 containerHeight - textHeight + sgOffsetY, textWidth, textHeight);
 
-            textSceneColorBrush.Color = black;
+            _textSceneColorBrush.Color = _black;
 
             //Draw title and x axis text.
```

Comment:
// comments need to have a space after the //

Reply:
updated